docs/docs/Components/components-logic.md

```diff
@@ -36,7 +36,52 @@ It includes code examples of REST and gRPC implementations to demonstrate integr
 
 ## Conditional router (If-Else component)
 
-This component routes messages by comparing two strings. It evaluates a condition by comparing two text inputs using the specified operator and routes the message to `true_result` or `false_result`.
+This component routes messages by comparing two strings.
+It evaluates a condition by comparing two text inputs using the specified operator and routes the message to `true_result` or `false_result`.
+
+The operator looks for single strings based on your defined [Operator behavior](#operator-behavior), but it can also search for multiple words by regex matching.
```

Comment:
```suggestion
The operator looks for single strings based on your defined [operator behavior](#operator-behavior), but it can also search for multiple words by regex matching.
```